CONTRIBUTING.md

```diff
@@ -80,6 +80,22 @@ Also be sure to skim through our [ADRs](https://github.com/backstage/backstage/t
 
 If there are any updates in `markdown` file please make sure to run `yarn run lint:docs`. Though it is checked on `lint-staged`. It is required to install [vale](https://docs.errata.ai/vale/install) separately and make sure it is accessed by global command.
 
+## DCO
```

Comment:
Scanning headers, I may not have known what this was - new users may not be as familiar. Could this be spelled out in the header, and then in the body (where you have already) you can list it and note it's abbreviation parenthetically first time before then using it further?